test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -260,6 +260,11 @@ Describe 'function statement parsing' -Tags "CI" {
 
 Describe 'assignment statement parsing' -Tags "CI" {
     ShouldBeParseError '$a,$b += 1,2' InvalidLeftHandSide 0
```

Comment:
Possibly worth putting in some strange cases here to make sure they fail in a sane way:

```powershell
$hello ??? $what
$cricket ?= $soccer
```

Reply:
Added